unitylibs/core/workflow/workflow-acrobat/upload-handler.js

```diff
@@ -522,7 +522,7 @@ export default class UploadHandler {
     try {
       const { blobDataArray, assetDataArray, fileTypeArray } = await this.createInitialAssets(files, workflowId, maxConcurrentFiles);
       if (assetDataArray.length === 0) {
-        await this.dispatchGenericError(`No assets created for the files: ${JSON.stringify(filesData)}`);
+        await this.actionBinder.dispatchErrorToast('pre_upload_error_create_asset', 500, `Error during asset creation or blob retrieval:`, false, true, {code: 'pre_upload_error_create_asset', desc: 'No assets created for the files, error is:',});
```

Comment:
The error message is using template literal but we are not passing any data to log. Should we just use quotes in this case?